File: frontend/src/constants/nativeConnections.ts
/**
 * Constants for native connections.
 * These MUST match the UUIDs defined in backend/airweave/core/constants/native_connections.py
 */

// Native connection UUIDs
export const NATIVE_QDRANT_UUID = "11111111-1111-1111-1111-111111111111";
const NATIVE_NEO4J_UUID = "22222222-2222-2222-2222-222222222222";
export const NATIVE_TEXT2VEC_UUID = "33333333-3333-3333-3333-333333333333";

// Map of connection short names to their UUIDs
const NATIVE_CONNECTION_UUIDS = {
  qdrant_native: NATIVE_QDRANT_UUID,
  neo4j_native: NATIVE_NEO4J_UUID,
  local_text2vec: NATIVE_TEXT2VEC_UUID,
};

File: backend/airweave/core/constants/native_connections.py
"""Constants for native connections."""

import uuid

# Native connection UUIDs - these must match the ones in init_db_native.py
NATIVE_WEAVIATE_UUID = uuid.UUID("11111111-1111-1111-1111-111111111111")
NATIVE_NEO4J_UUID = uuid.UUID("22222222-2222-2222-2222-222222222222")
NATIVE_TEXT2VEC_UUID = uuid.UUID("33333333-3333-3333-3333-333333333333")

# String versions for use in frontend code or string contexts
NATIVE_WEAVIATE_UUID_STR = str(NATIVE_WEAVIATE_UUID)
NATIVE_NEO4J_UUID_STR = str(NATIVE_NEO4J_UUID)
NATIVE_TEXT2VEC_UUID_STR = str(NATIVE_TEXT2VEC_UUID)
